### ISSUE TITLE
Missing Dependencies for paket installation

### ISSUE THREAD

**Issue by Allandor** [Icinga/icinga2#5442]
## Current Behavior

Problem with installing icinga2 package during OS installation.
Log output:
```
2017-07-26 15:34:44 icinga2-common-2.6.3-1.1.x86_64.rpm installed ok
# Additional rpm output:
# /var/tmp/rpm-tmp.V659IY: line 1: /usr/sbin/groupadd: No such file or directory
# /var/tmp/rpm-tmp.V659IY: line 2: /usr/sbin/groupadd: No such file or directory
# /var/tmp/rpm-tmp.V659IY: line 3: /usr/sbin/useradd: No such file or directory
# /var/tmp/rpm-tmp.V659IY: line 8: mkdir: command not found
# /var/tmp/rpm-tmp.V659IY: line 12: touch: command not found

```
## Possible Solution

Add dependency for package "shadow" and "coreutils"
```
rpm -qf /usr/sbin/useradd
shadow-4.2.1-6.3.1.x86_64
rpm -qf /usr/bin/mkdir
coreutils-8.25-1.4.x86_64
rpm -qf /usr/bin/touch 
coreutils-8.25-1.4.x86_64
```

## Your Environment
OpenSuSE 42.2

```
Informationen zu Paket icinga2:
-------------------------------
Repository         : [OpenSuSE 42.2] Monitoring                
Name               : icinga2                                   
Version            : 2.6.3-1.1                                 
Arch               : x86_64                                    
Anbieter           : obs://build.opensuse.org/server:monitoring
Installierte Größe : 17,6 KiB                                  
Installiert        : Ja                                        
Status             : aktuell                                   
Quellpaket         : icinga2-2.6.3-1.1.src                     
Zusammenfassung    : Network monitoring application            
Beschreibung       :                                           
    Meta package for Icinga 2 Core, DB IDO and Web.
```

**Comment by Crunsher** [comment]
Are you missing the commands mkdir and touch as well?

**Comment by Allandor** [comment]
Yes. I updated the issue.
```
rpm -qf /usr/bin/mkdir
coreutils-8.25-1.4.x86_64
```

**Comment by dgoetz** [comment]
I have no idea why we have:

```
%if "%{_vendor}" == "redhat"
Requires(pre): shadow-utils
Requires(post): shadow-utils
%endif
```

but no:

```
%if "%{_vendor}" == "suse"
Requires(pre): shadow
Requires(post): shadow
%endif
```

But why is coreutils not installed? I would assume this being installed as a valid default.

**Comment by Allandor** [comment]
Icinga2 was added to the package list of autoyast. So it is possible that coreutils will be installed afterwards. During some tests this could be replicated. So a dependency could fix this.

**Comment by lazyfrosch** [comment]
Do we need coreutils or is this covered by shadow?

**Comment by Allandor** [comment]
It shold be covered by shadow, with the permissions package.

**Comment by lazyfrosch** [comment]
@dgoetz wanted to do a PR against https://github.com/icinga/icinga-packaging

**Comment by dgoetz** [comment]
PR is there, please test build against Suse as I only could test I do not break EPEL and Fedora.

**Comment by lazyfrosch** [comment]
This has been resolved in the snapshot packages, and will be released with Icinga 2.7.0-1 tomorrow.
